performance/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "ssh://git@github.com/k6io/example-typescript.git",
+  "author": "Simon Aronsson <simme@k6.io>",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "7.13.16",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/preset-env": "7.13.15",
+    "@babel/preset-typescript": "7.13.0",
+    "@types/k6": "^0.45.3",
+    "@types/webpack": "5.28.0",
+    "babel-loader": "8.2.2",
+    "clean-webpack-plugin": "4.0.0-alpha.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "typescript": "4.2.4",
+    "webpack": "5.76.1",
+    "webpack-cli": "5.0.1",
+    "webpack-glob-entries": "^1.0.1"
```

Comment:
given that the webui uses vite/esbuild and this has us using webpack/babel, consider tagging some work to migrate this to vite using library mode: https://vitejs.dev/guide/build.html#library-mode

Reply:
ticket here: https://hpe-aiatscale.atlassian.net/browse/WEB-1623